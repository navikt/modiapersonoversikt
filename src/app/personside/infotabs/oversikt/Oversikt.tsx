import * as React from 'react';
import styled from 'styled-components/macro';
import Oversiktskomponent from './Oversiktskomponent';
import theme, { pxToRem } from '../../../../styles/personOversiktTheme';
import VarselOversikt from './VarselOversikt';
import OppfolgingOversikt from './OppfolgingOversikt';
import YtelserOversikt from './YtelserOversikt';
import UtbetalingerOversikt from './UtbetalingerOversikt';
import MeldingerOversikt from './MeldingerOversikt';
import { INFOTABS } from '../InfoTabEnum';
import ErrorBoundary from '../../../../components/ErrorBoundary';
import SakerOversiktFactory from './SakerOversiktFactory';

const oversiktMediaThreshold = pxToRem(750);

const Style = styled.div`
    @media (min-width: ${oversiktMediaThreshold}) {
        display: flex;
        justify-content: center;
    }
    padding: ${theme.margin.layout};
    .alertstripe {
        background-color: transparent;
        border: none;
    }
`;

const KolonneStyle = styled.div`
    > * {
        margin-bottom: ${theme.margin.layout};
    }
    @media (min-width: ${oversiktMediaThreshold}) {
        &:not(:last-child) {
            margin-right: ${theme.margin.layout};
        }
    }
    flex-basis: 50%;
    max-width: ${pxToRem(800)};
`;

function Oversikt() {
    return (
        <ErrorBoundary boundaryName="Oversikt">
            <Style>
                <KolonneStyle>
                    <Oversiktskomponent
                        infotabPath={INFOTABS.OPPFOLGING}
                        tittel={'OppfÃ¸lging'}
                        component={OppfolgingOversikt}
                        hurtigtast={'T'}
                    />
                    <Oversiktskomponent
                        infotabPath={INFOTABS.UTBETALING}
                        tittel={'Utbetalinger'}
                        component={UtbetalingerOversikt}
                        hurtigtast={'U'}
                    />
                    <Oversiktskomponent
                        tittel={'Saker'}
                        infotabPath={INFOTABS.SAKER}
                        component={SakerOversiktFactory}
                        hurtigtast={'S'}
                    />
                </KolonneStyle>
                <KolonneStyle>
                    <Oversiktskomponent
                        infotabPath={INFOTABS.MELDINGER}
                        tittel={'Kommunikasjon'}
                        component={MeldingerOversikt}
                        hurtigtast={'M'}
                    />
                    <Oversiktskomponent
                        infotabPath={INFOTABS.YTELSER}
                        tittel={'Ytelser'}
                        component={YtelserOversikt}
                        hurtigtast={'Y'}
                    />
                    <Oversiktskomponent
                        infotabPath={INFOTABS.VARSLER}
                        tittel={'Varsler'}
                        component={VarselOversikt}
                        hurtigtast={'V'}
                    />
                </KolonneStyle>
            </Style>
        </ErrorBoundary>
    );
}

export default Oversikt;
