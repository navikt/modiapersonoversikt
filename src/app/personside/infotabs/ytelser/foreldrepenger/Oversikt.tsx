import * as React from 'react';
import { Foreldrepengerettighet } from '../../../../../models/ytelse/foreldrepenger';
import { utledFraDatoForRettighet, utledMaksDato } from './foreldrePengerUtils';
import { datoVerbose } from '../../utbetalinger/utils/utbetalingerUtils';
import DescriptionList from '../felles-styling/DescriptionList';
import YtelserBullet from '../felles-styling/YtelserBullet';
import { OversiktStyling } from '../felles-styling/CommonStylingYtelser';

interface Props {
    foreldrePenger: Foreldrepengerettighet;
}

function Oversikt({foreldrePenger}: Props) {

    const foreldrePengeRetten = {
        Foreldrepengetype: foreldrePenger.foreldrepengetype,
        Dekningsgrad: foreldrePenger.dekningsgrad + '%',
        'Rettighet fra dato': datoVerbose(utledFraDatoForRettighet(foreldrePenger)).sammensatt,
        Restdager: foreldrePenger.restDager,
        Maksdato: utledMaksDato(foreldrePenger),
        Arbeidskategori: foreldrePenger.arbeidskategori
    };

    const barnet = {
        Termindato: foreldrePenger.rettighetFom && datoVerbose(foreldrePenger.rettighetFom).sammensatt,
        Fødselsdato: foreldrePenger.barnetsFødselsdato && datoVerbose(foreldrePenger.barnetsFødselsdato).sammensatt,
        'Annen forelder': foreldrePenger.andreForeldersFnr,
        Omsorgsovertakelse: 'Ikke implementert',
        'Foreldre av samme kjønn': 'Ikke implementert',
        'Antall barn': foreldrePenger.antallBarn
    };

    return (
        <OversiktStyling>
            <YtelserBullet tittel="Om foreldrepengeretten">
                <DescriptionList entries={foreldrePengeRetten}/>
            </YtelserBullet>
            <YtelserBullet tittel="Om barnet">
                <DescriptionList entries={barnet}/>
            </YtelserBullet>
        </OversiktStyling>
    );
}

export default Oversikt;
