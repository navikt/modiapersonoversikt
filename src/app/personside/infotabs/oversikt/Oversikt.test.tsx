import { render } from '@testing-library/react';
import TestProvider from '../../../../test/Testprovider';
import { setupReactQueryMocks } from '../../../../test/testStore';
import Oversikt from './Oversikt';

test('Viser oversikt med alt innhold', () => {
    setupReactQueryMocks();
    const container = render(
        <TestProvider>
            <Oversikt />
        </TestProvider>
    );

    const json = container.asFragment();
    expect(json).toMatchSnapshot();
    container.unmount();
});
