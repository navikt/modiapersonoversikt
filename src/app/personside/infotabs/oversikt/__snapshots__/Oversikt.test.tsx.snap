// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Viser oversikt med alt innhold 1`] = `
.c2 {
  border-radius: 0.25rem;
  background-color: white;
  box-shadow: 0 0 0.5rem rgba(0,0,0,0.1),0 0 0 1px rgba(0,0,0,0.1);
  padding-bottom: 0.3rem;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  background-color: white;
  border-top-left-radius: .25rem;
  border-top-right-radius: .25rem;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  padding: 0.625rem;
  cursor: pointer;
  border-bottom: #b7b1a9 0.125rem solid;
}

.c3:hover {
  box-shadow: inset 0 0 0 0.1rem #0067c5,0 0.1rem 0.1rem rgba(0,0,0,0.2);
}

.c7 > *:not(:first-child) {
  border-top: solid 0.0625rem #78706a;
}

.c6 {
  text-align: right;
}

.c5 {
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  display: -webkit-inline-box;
  display: -webkit-inline-flex;
  display: -ms-inline-flexbox;
  display: inline-flex;
  -webkit-align-items: flex-end;
  -webkit-box-align: flex-end;
  -ms-flex-align: flex-end;
  align-items: flex-end;
  padding: 0 1rem;
}

.c4 {
  font-weight: bold !important;
  font-size: 1.125rem !important;
}

.c31 {
  padding: 0.7rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c31 > *:first-child {
  -webkit-flex: 0 0 20%;
  -ms-flex: 0 0 20%;
  flex: 0 0 20%;
}

.c31 > *:last-child {
  max-width: 30rem;
}

.c30 {
  border-radius: 0.5rem;
  background-color: #f1f0f0;
  box-shadow: inset 0 0 0 0.3rem white,inset 0 0 0 0.37rem rgba(0,0,0,0.15);
  padding: 1.25rem;
}

.c30 > *:not(:first-child) {
  border-top: solid 0.0625rem #78706a;
}

.c29 {
  border: none;
  cursor: pointer;
  background-color: transparent;
  padding: 0.1rem;
  border-radius: 0.5em;
}

.c29:focus {
  outline: none;
  box-shadow: 0 0 0 0.1875rem #254b6d;
}

.c29:focus {
  outline: none;
  box-shadow: none;
}

.c29:focus::after {
  position: absolute;
  top: 0;
  left: 0;
  content: '';
  box-shadow: inset 0 0 0 0.1875rem #254b6d;
  height: 100%;
  width: 100%;
  pointer-events: none;
}

.c26 {
  border-radius: 0.25rem;
  background-color: white;
  box-shadow: 0 0 0.5rem rgba(0,0,0,0.1),0 0 0 1px rgba(0,0,0,0.1);
}

.c27 {
  position: relative;
  display: -ms-grid;
  display: grid;
  -ms-grid-columns: 6rem 55% 1fr auto;
  grid-template-columns: 6rem minmax(35%,55%) 1fr auto;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  border-radius: .25rem;
  cursor: pointer;
}

.c27 > *:nth-child(1) {
  -ms-grid-column: 1;
}

.c27 > *:nth-child(2) {
  -ms-grid-column: 2;
}

.c27 > *:nth-child(3) {
  -ms-grid-column: 3;
}

.c27 > *:nth-child(4) {
  -ms-grid-column: 4;
}

.c27:hover {
  box-shadow: inset 0 0 0 0.1rem #0067c5,0 0.1rem 0.1rem rgba(0,0,0,0.2);
}

.c27 > * {
  padding: 0.7rem;
}

.c28 li {
  display: inline-block;
}

.c28 li:not(:last-child):after {
  content: ',';
  margin-right: 0.5em;
}

.c25 > *:not(:first-child) {
  border-top: solid 0.0625rem #78706a;
}

.c9 {
  position: relative;
  padding: .5rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  cursor: pointer;
}

.c9:hover {
  box-shadow: inset 0 0 0 0.1rem #0067c5,0 0.1rem 0.1rem rgba(0,0,0,0.2);
}

.c9:hover > button:last-child {
  padding-left: 0.5rem;
}

.c9 > *:first-child {
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

.c11 {
  border: none;
  padding: 0;
  height: 2rem;
  width: 2rem;
  border-radius: .35rem;
  cursor: pointer;
  background-color: transparent;
  -webkit-transition: padding-left 0.3s;
  transition: padding-left 0.3s;
}

.c11 .nav-frontend-chevron {
  width: 1.5625rem;
}

.c11:focus {
  outline: none;
  box-shadow: none;
}

.c11:focus::after {
  position: absolute;
  top: 0;
  left: 0;
  content: '';
  box-shadow: inset 0 0 0 0.1875rem #254b6d;
  height: 100%;
  width: 100%;
  pointer-events: none;
}

.c15 svg {
  height: 1.875rem;
  width: 1.875rem;
  opacity: 0.5;
}

.c14 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c14 .order-first {
  -webkit-order: 0;
  -ms-flex-order: 0;
  order: 0;
}

.c14 .order-second {
  -webkit-order: 1;
  -ms-flex-order: 1;
  order: 1;
}

.c13 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
}

.c24 > *:not(:first-child) {
  border-top: solid 0.0625rem #78706a;
}

.c10 {
  font-weight: 600;
}

.c8 > *:not(:first-child) {
  border-top: solid 0.0625rem #78706a;
}

.c18 {
  position: relative;
}

.c18 svg {
  height: 1.5625rem;
  width: 1.5625rem;
  opacity: 0.4;
}

.c19 {
  position: absolute;
  top: -0.1875rem;
  right: -0.5rem;
  background-color: #78706a;
  color: white;
  height: 1.1rem;
  width: 1.1rem;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  border-radius: 50%;
}

.c20 {
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
  width: 0;
  margin-left: 0.8rem;
  overflow-wrap: break-word;
}

.c23 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  margin-top: 0.2rem;
}

.c23 > *:not(:last-child) {
  margin-bottom: 0.2rem;
  margin-right: 0.2rem;
}

.c21 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
}

.c21 .order-first {
  -webkit-order: 0;
  -ms-flex-order: 0;
  order: 0;
}

.c21 .order-second {
  -webkit-order: 1;
  -ms-flex-order: 1;
  order: 1;
}

.c17 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
}

.c17 > *:last-child {
  -webkit-box-flex: 1;
  -webkit-flex-grow: 1;
  -ms-flex-positive: 1;
  flex-grow: 1;
}

.c22 {
  text-overflow: ellipsis;
  overflow: hidden;
  white-space: nowrap;
}

.c16 > *:not(:first-child) {
  border-top: solid 0.0625rem #78706a;
}

.c12 > *:not(:first-child) {
  border-top: solid 0.0625rem #78706a;
}

.c0 {
  padding: .5rem;
}

.c0 .alertstripe {
  background-color: transparent;
  border: none;
}

.c1 {
  -webkit-flex-basis: 50%;
  -ms-flex-preferred-size: 50%;
  flex-basis: 50%;
  max-width: 50rem;
}

.c1 > * {
  margin-bottom: .5rem;
}

@media (min-width:35rem) {
  .c0 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-box-pack: center;
    -webkit-justify-content: center;
    -ms-flex-pack: center;
    justify-content: center;
  }
}

@media (min-width:35rem) {
  .c1:not(:last-child) {
    margin-right: .5rem;
  }
}

<div
  className="c0"
>
  <div
    className="c1"
  >
    <article
      aria-describedby="Helt tilfeldig ID"
      className="c2"
    >
      <div
        className="c3"
        onClick={[Function]}
        title="Alt + T"
      >
        <h3
          className="typo-undertittel c4"
          id="Helt tilfeldig ID"
        >
          Oppfølging
        </h3>
        <div
          className="c5"
        />
        <a
          className="lenke c6"
          href="/modiapersonoversikt/person/10108000398/oppfølging/"
          onClick={[Function]}
        >
          <p
            className="typo-normal"
          >
            Gå til 
            oppfølging
          </p>
        </a>
      </div>
      <div
        className="c7"
      >
        <div
          className="alertstripe alertstripe--info"
        >
          <span
            aria-label="info"
            className="alertstripe__ikon"
          >
            <svg
              focusable="false"
              height="1.5em"
              kind="info-sirkel-fyll"
              viewBox="0 0 24 24"
              width="1.5em"
            >
              <g
                fill="none"
              >
                <path
                  d="M12 0C5.382 0 0 5.382 0 12s5.382 12 12 12c6.617 0 12-5.382 12-12S18.617 0 12 0z"
                  fill="#669DB4"
                />
                <path
                  d="M12 5a1.566 1.566 0 1 1 .11 3.13A1.566 1.566 0 0 1 12 5zm2.976 12.01c.563 0 1.043.431 1.043.991s-.48.992-1.043.992H9.39c-.564 0-1.043-.431-1.043-.992 0-.56.479-.99 1.043-.99h1.6v-5.016h-.986c-.565 0-1.044-.43-1.044-.991 0-.56.48-.991 1.044-.991h2.03c.563 0 1.043.43 1.043.99v6.007h1.899z"
                  fill="#FFF"
                />
              </g>
            </svg>
          </span>
          <div
            className="typo-normal alertstripe__tekst"
          >
            Er ikke i arbeidsrettet oppfølging
          </div>
        </div>
      </div>
    </article>
    <article
      aria-describedby="Helt tilfeldig ID"
      className="c2"
    >
      <div
        className="c3"
        onClick={[Function]}
        title="Alt + U"
      >
        <h3
          className="typo-undertittel c4"
          id="Helt tilfeldig ID"
        >
          Utbetalinger
        </h3>
        <div
          className="c5"
        />
        <a
          className="lenke c6"
          href="/modiapersonoversikt/person/10108000398/utbetaling/"
          onClick={[Function]}
        >
          <p
            className="typo-normal"
          >
            Gå til 
            utbetalinger
          </p>
        </a>
      </div>
      <div
        className="c7"
      >
        <ol
          aria-label="Oversikt brukers utbetalinger"
          className="c8"
        >
          <li
            className="test-utbetalinger-oversikt"
          >
            <div
              className="c9"
              onClick={[Function]}
            >
              <div>
                <p
                  className="typo-normal"
                >
                  05.05.2019
                   / 
                  Ligger hos banken
                </p>
                <p
                  className="typo-normal"
                >
                  <span
                    className="c10"
                  >
                    Barnetrygd
                  </span>
                </p>
                <p
                  className="typo-normal"
                >
                  13.02.2018
                   - 
                  20.09.2018
                </p>
                <p
                  className="typo-normal"
                >
                  Utbetaling til: 
                  AREMARK TESTFAMILIEN
                </p>
              </div>
              <button
                aria-label="Vis utbetaling"
                aria-selected={false}
                className="c11"
              >
                <i
                  className="nav-frontend-chevron chevronboks chevron--hoyre"
                />
              </button>
            </div>
          </li>
          <li
            className="test-utbetalinger-oversikt"
          >
            <div
              className="c9"
              onClick={[Function]}
            >
              <div>
                <p
                  className="typo-normal"
                >
                  17.12.2018
                   / 
                  Ligger hos banken
                </p>
                <p
                  className="typo-normal"
                >
                  <span
                    className="c10"
                  >
                    Diverse ytelser
                  </span>
                </p>
                <p
                  className="typo-normal"
                >
                  12.05.2018
                   - 
                  10.04.2019
                </p>
                <p
                  className="typo-normal"
                >
                  Utbetaling til: 
                  AREMARK TESTFAMILIEN
                </p>
              </div>
              <button
                aria-label="Vis utbetaling"
                aria-selected={false}
                className="c11"
              >
                <i
                  className="nav-frontend-chevron chevronboks chevron--hoyre"
                />
              </button>
            </div>
          </li>
        </ol>
      </div>
    </article>
    <article
      aria-describedby="Helt tilfeldig ID"
      className="c2"
    >
      <div
        className="c3"
        onClick={[Function]}
        title="Alt + S"
      >
        <h3
          className="typo-undertittel c4"
          id="Helt tilfeldig ID"
        >
          Saker
        </h3>
        <div
          className="c5"
        />
        <a
          className="lenke c6"
          href="/modiapersonoversikt/person/10108000398/saker/"
          onClick={[Function]}
        >
          <p
            className="typo-normal"
          >
            Gå til 
            saker
          </p>
        </a>
      </div>
      <div
        className="c7"
      >
        <ol
          className="c12"
        >
          <li
            className="test-sakstema-list"
          >
            <div
              className="c9"
              onClick={[Function]}
            >
              <div>
                <div
                  className="c13"
                >
                  <div>
                    <div
                      className="c14"
                    >
                      <p
                        className="typo-element order-second test-saker-oversikt"
                      >
                        Sykepenger
                      </p>
                      <p
                        className="typo-normal order-first"
                      >
                        24.12.2018
                      </p>
                    </div>
                    <p
                      className="typo-normal"
                    >
                      1
                       
                      søknad
                       er 
                      under behandling
                      .
                    </p>
                  </div>
                  <span
                    className="c15"
                  >
                    <svg
                      contentScriptType="text/ecmascript"
                      viewBox="0 0 24 24"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <g
                        fill="currentColor"
                      >
                        <path
                          d="M17.5 10c1.3 0 2.5.3 3.5.9V4.5c0-.3-.2-.5-.5-.5H19V2.5c0-.3-.2-.5-.5-.5H6V.5c0-.3-.2-.5-.5-.5h-5C.2 0 0 .2 0 .5V16c0 1.1.9 2 2 2h8v-.5c0-4.1 3.4-7.5 7.5-7.5zM3 4.5V16a1 1 0 0 1-2 0V1h4v1.5c0 .3.2.5.5.5H18v1H3.5c-.3 0-.5.2-.5.5z"
                        />
                        <path
                          d="M17.5 11a6.5 6.5 0 0 0-5 10.7l9.2-9.1a6.5 6.5 0 0 0-4.2-1.6zM22.4 13.3l-9.1 9.1c1.1 1 2.6 1.6 4.2 1.6a6.5 6.5 0 0 0 5-10.7z"
                        />
                      </g>
                    </svg>
                  </span>
                </div>
              </div>
              <button
                aria-label="Vis Sykepenger"
                aria-selected={false}
                className="c11"
              >
                <i
                  className="nav-frontend-chevron chevronboks chevron--hoyre"
                />
              </button>
            </div>
          </li>
          <li
            className="test-sakstema-list"
          >
            <div
              className="c9"
              onClick={[Function]}
            >
              <div>
                <div
                  className="c13"
                >
                  <div>
                    <div
                      className="c14"
                    >
                      <p
                        className="typo-element order-second test-saker-oversikt"
                      >
                        Arbeidsavklaringspenger
                      </p>
                      <p
                        className="typo-normal order-first"
                      >
                        25.06.2018
                      </p>
                    </div>
                    <p
                      className="typo-normal"
                    >
                      1
                       
                      søknad
                       er 
                      under behandling
                      .
                    </p>
                    <p
                      className="typo-normal"
                    >
                      2
                       
                      søknader
                       er 
                      ferdig behandlet
                      .
                    </p>
                  </div>
                  <span
                    className="c15"
                  />
                </div>
              </div>
              <button
                aria-label="Vis Arbeidsavklaringspenger"
                aria-selected={false}
                className="c11"
              >
                <i
                  className="nav-frontend-chevron chevronboks chevron--hoyre"
                />
              </button>
            </div>
          </li>
        </ol>
      </div>
    </article>
  </div>
  <div
    className="c1"
  >
    <article
      aria-describedby="Helt tilfeldig ID"
      className="c2"
    >
      <div
        className="c3"
        onClick={[Function]}
        title="Alt + M"
      >
        <h3
          className="typo-undertittel c4"
          id="Helt tilfeldig ID"
        >
          Meldinger
        </h3>
        <div
          className="c5"
        />
        <a
          className="lenke c6"
          href="/modiapersonoversikt/person/10108000398/meldinger/"
          onClick={[Function]}
        >
          <p
            className="typo-normal"
          >
            Gå til 
            meldinger
          </p>
        </a>
      </div>
      <div
        className="c7"
      >
        <ol
          aria-label="Oversikt brukers meldinger"
          className="c16"
        >
          <li>
            <div
              className="test-meldinger-oversikt c9"
              onClick={[Function]}
            >
              <div>
                <div
                  className="c17"
                >
                  <span
                    className="c18"
                  >
                    <svg
                      viewBox="0 0 24 24"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <g
                        fill="#3e3832"
                      >
                        <path
                          d="M16.5 8.75c.476 0 .941.045 1.399.115.065-.379.101-.771.101-1.18C18 3.585 13.962.25 9 .25c-4.963 0-9 3.335-9 7.435 0 1.962.855 3.826 2.36 5.178l-1.794 3.14a.5.5 0 00.62.712l4.839-1.937c.435.124.887.218 1.348.281.022.002.065-.005.075.004.075 0 .319-.002.678-.022.679-3.514 4.233-6.291 8.374-6.291z"
                        />
                        <path
                          d="M16.5 9.75c-4.065 0-7.5 2.977-7.5 6.5 0 3.355 2.878 6.224 6.434 6.572a7.581 7.581 0 003.578-.534l3.813 1.43a.498.498 0 00.623-.691l-1.325-2.65C23.327 19.289 24 17.868 24 16.25c0-3.523-3.435-6.5-7.5-6.5z"
                        />
                      </g>
                    </svg>
                    <p
                      aria-hidden="true"
                      className="typo-undertekst c19"
                    >
                      4
                    </p>
                  </span>
                  <div
                    className="c20"
                  >
                    <div
                      className="c21"
                    >
                      <p
                        aria-hidden={true}
                        className="typo-element order-second"
                      >
                        Delsvar - Arbeid
                      </p>
                      <p
                        className="typo-normal order-first"
                      >
                        27.07.2019 00:00
                      </p>
                    </div>
                    <p
                      className="typo-normal c22"
                    >
                      Dolores omnis ea voluptatem corrupti odit incidunt enim. Unde fugiat sed animi explicabo maiores. Nihil cum accusamus omnis officiis deserunt est. Qui quisquam unde aut magni dignissimos.
                    </p>
                    <div
                      className="c23"
                    >
                      <div
                        className="etikett etikett--fokus"
                      >
                        <span
                          className="typo-normal"
                        >
                          Under arbeid
                        </span>
                      </div>
                      <div
                        className="etikett etikett--info"
                      >
                        <span
                          className="typo-normal"
                        >
                          Delvis besvart
                        </span>
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <button
                aria-label="Vis meldinger for Delsvar - Arbeid"
                aria-selected={false}
                className="c11"
              >
                <i
                  className="nav-frontend-chevron chevronboks chevron--hoyre"
                />
              </button>
            </div>
          </li>
        </ol>
      </div>
    </article>
    <article
      aria-describedby="Helt tilfeldig ID"
      className="c2"
    >
      <div
        className="c3"
        onClick={[Function]}
        title="Alt + Y"
      >
        <h3
          className="typo-undertittel c4"
          id="Helt tilfeldig ID"
        >
          Ytelser
        </h3>
        <div
          className="c5"
        />
        <a
          className="lenke c6"
          href="/modiapersonoversikt/person/10108000398/ytelser/"
          onClick={[Function]}
        >
          <p
            className="typo-normal"
          >
            Gå til 
            ytelser
          </p>
        </a>
      </div>
      <div
        className="c7"
      >
        <div
          className="c24"
        >
          <div
            className="test-ytelse-oversikt c9"
            onClick={[Function]}
          >
            <div>
              <p
                className="typo-normal"
              >
                ID dato: 
                06.02.2019
              </p>
              <p
                className="typo-element"
              >
                Sykepenger
              </p>
              <p
                className="typo-normal"
              >
                100% sykemeldt - Maksdato
                 
                16.01.2019
              </p>
            </div>
            <button
              aria-label="Vis sykepenger"
              aria-selected={false}
              className="c11"
            >
              <i
                className="nav-frontend-chevron chevronboks chevron--hoyre"
              />
            </button>
          </div>
          <div
            className="test-ytelse-oversikt c9"
            onClick={[Function]}
          >
            <div>
              <p
                className="typo-element"
              >
                Pleiepenger sykt barn
              </p>
              <p
                className="typo-normal"
              >
                Barnets f.nr: 
                08031815240
              </p>
            </div>
            <button
              aria-label="Vis pleiepenger"
              aria-selected={false}
              className="c11"
            >
              <i
                className="nav-frontend-chevron chevronboks chevron--hoyre"
              />
            </button>
          </div>
        </div>
      </div>
    </article>
    <article
      aria-describedby="Helt tilfeldig ID"
      className="c2"
    >
      <div
        className="c3"
        onClick={[Function]}
        title="Alt + V"
      >
        <h3
          className="typo-undertittel c4"
          id="Helt tilfeldig ID"
        >
          Varsler
        </h3>
        <div
          className="c5"
        />
        <a
          className="lenke c6"
          href="/modiapersonoversikt/person/10108000398/varsler/"
          onClick={[Function]}
        >
          <p
            className="typo-normal"
          >
            Gå til 
            varsler
          </p>
        </a>
      </div>
      <div
        className="c7"
      >
        <ol
          className="c25"
        >
          <li
            className="c26"
          >
            <article
              aria-describedby="Helt tilfeldig ID"
            >
              <div
                className="c27"
                onClick={[Function]}
              >
                <p
                  className="typo-normal"
                >
                  28.03.2019
                </p>
                <h4
                  className="typo-element"
                  id="Helt tilfeldig ID"
                >
                  Påminnelse om å sende meldekort
                </h4>
                <ul
                  aria-label="Kommunikasjonskanaler"
                  className="c28"
                >
                  <li
                    className="typo-normal"
                  >
                    EPOST
                  </li>
                  <li
                    className="typo-normal"
                  >
                    SMS
                  </li>
                </ul>
                <button
                  aria-expanded={false}
                  aria-label="Vis meldinger"
                  className="c29"
                  onClick={[Function]}
                  title="Vis meldinger"
                >
                  <i
                    className="nav-frontend-chevron chevronboks chevron--ned"
                  />
                </button>
              </div>
              <ol
                aria-label="Meldinger"
                className="c30"
              >
                <li
                  aria-label="EPOST"
                  className="c31"
                >
                  <div>
                    <p
                      className="typo-element"
                    >
                      EPOST
                    </p>
                    <p
                      className="typo-normal"
                    >
                      01. mars 2019
                    </p>
                  </div>
                  <div>
                    <div>
                      <p
                        className="typo-element"
                      >
                        Epostemne
                      </p>
                      <p
                        className="typo-normal"
                      >
                        Meldingsinnhold
                      </p>
                    </div>
                    <p
                      className="typo-element"
                    >
                      Epost: 
                      mottakerinfo
                    </p>
                  </div>
                </li>
                <li
                  aria-label="SMS"
                  className="c31"
                >
                  <div>
                    <p
                      className="typo-element"
                    >
                      SMS
                    </p>
                    <p
                      className="typo-normal"
                    >
                      24. jan. 2019
                    </p>
                  </div>
                  <div>
                    <p
                      className="typo-normal"
                    >
                      Meldingsinnhold
                    </p>
                    <p
                      className="typo-element"
                    >
                      Tlf: 
                      mottakerinfo
                    </p>
                  </div>
                </li>
              </ol>
            </article>
          </li>
          <li
            className="c26"
          >
            <article
              aria-describedby="Helt tilfeldig ID"
            >
              <div
                className="c27"
                onClick={[Function]}
              >
                <p
                  className="typo-normal"
                >
                  10.01.2019
                </p>
                <h4
                  className="typo-element"
                  id="Helt tilfeldig ID"
                >
                  Påminnelse om å sende meldekort
                </h4>
                <ul
                  aria-label="Kommunikasjonskanaler"
                  className="c28"
                >
                  <li
                    className="typo-normal"
                  >
                    EPOST
                  </li>
                </ul>
                <button
                  aria-expanded={false}
                  aria-label="Vis meldinger"
                  className="c29"
                  onClick={[Function]}
                  title="Vis meldinger"
                >
                  <i
                    className="nav-frontend-chevron chevronboks chevron--ned"
                  />
                </button>
              </div>
              <ol
                aria-label="Meldinger"
                className="c30"
              >
                <li
                  aria-label="EPOST"
                  className="c31"
                >
                  <div>
                    <p
                      className="typo-element"
                    >
                      EPOST
                    </p>
                    <p
                      className="typo-normal"
                    >
                      02. feb. 2019
                    </p>
                  </div>
                  <div>
                    <div>
                      <p
                        className="typo-element"
                      >
                        Epostemne
                      </p>
                      <p
                        className="typo-normal"
                      >
                        Meldingsinnhold
                      </p>
                    </div>
                    <p
                      className="typo-element"
                    >
                      Epost: 
                      mottakerinfo
                    </p>
                  </div>
                </li>
              </ol>
            </article>
          </li>
        </ol>
      </div>
    </article>
  </div>
</div>
`;
